playterminal/settings/base.py

```diff
@@ -101,3 +101,10 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Try to get the secrets values from the environment
+secret_vars = ['SECRET_KEY', 'STEPIC_CLIENT_ID', 'STEPIC_CLIENT_SECRET']
+for var in secret_vars:
+    value = os.getenv(var)
```

Comment:
The implemented approach doesn't exclude secret files and used specifically for CircleCI.
I also prefer secret files, yes.